packages/admin/cms-admin/src/blocks/createRichTextBlock.tsx

```diff
@@ -175,6 +175,19 @@ export const createRichTextBlock = (
             };
         },
 
+        extractTextContents: (state) => {
+            return transformStateToXml(state.editorState.getCurrentContent());
+        },
+
+        replaceTextContents: (state, contents) => {
+            const { editorState } = state;
+            const rawContent = convertStateToRawContent(editorState);
+
+            return {
+                editorState: translateAndTransformXmlToState(state.editorState.getCurrentContent(), rawContent, contents),
```

Comment:
Shouldn't the translation only happen here, and not in the `@comet/admin-rte` package?

Reply:
i had in mind that another comment said that it should not be in the cms-admin package, but I can move it back here

Reply:
Sorry, I wasn't very clear then: xmlToState and stateToXml should be in the `@comet/admin-rte` package, where they belong. But the translation feature is a CMS-only feature that is not required in the RTE package. Would it be possible to split translating and transforming?